bin/xucli

```diff
@@ -2,6 +2,9 @@
 
 require('yargs')
   .options({
+    rpc: {
+      default: {},
```

Comment:
Change this line to `hidden: true,` and I think this gives us the desirable behavior.